packages/commonwealth/server/routes/topics/create_topic_handler.ts

```diff
@@ -13,7 +19,28 @@ export const createTopicHandler = async (
 ) => {
   const { user, chain, body } = req;
 
-  const topic = await controllers.topics.createTopic({ user, chain, body });
+  const validationSchema = z.object({
```

Comment:
Also found out just now about:
https://www.npmjs.com/package/@anatine/zod-mock

I know you said you would be writing unit tests. I think this would be great to introduce, as we can easily mock the input type. Along with unit_helpers.ts we will be able to create unit tests efficiently and in a way that is robust to schema changes.